vnext/Microsoft.ReactNative.Cxx/UI.Text.h

```diff
@@ -3,8 +3,11 @@
 
 #pragma once
 
+#include <winrt/Windows.UI.Text.h>
+
 #ifdef USE_WINUI3
 #include <winrt/Microsoft.UI.Text.h>
```

Comment:
Crazy thought:
Should WinUI3 produce an entire compatibility alias-layer (perhaps both directions) for folks that need to support both?

Reply:
That's a noble idea :) some types/APIs don't exist in one or the other so it's not a clean rename, unfortunately. That's why you'll see blocks of code in our files with #ifdef USE_WINUI3 or #ifndef USE_WINUI3. There's more work to do to support what will be missing features in winui3 at launch (e.g. core window keyboard occlusion events, etc.)

Reply:
@dannyvv there is however an intern working on a conversion tool. I think the expectation is that most people just migrate once and they're done and don't have to keep supporting both, we're the odd duck here